Add ASP.NET samples and update readme

danroth27 opened the pull request:
The ASP.NET samples that used to live on CodePlex at https://aspnet.codeplex.com have now been archived as a zip file. All of the sample links on https://asp.net point to this archive page.

The proposal is to move all of these ASP.NET samples here and update the links on https://asp.net to point to the new locations in this repo.

Eilon commented on README.md:
```diff
@@ -1 +1,7 @@
+
+Samples for ASP.NET and ASP.NET.
+
+ASP.NET samples are under the *samples/aspnet* directory.
```
Can you make these links?

Eilon commented on README.md:
```diff
@@ -1 +1,7 @@
 # Samples
+
+Samples for ASP.NET and ASP.NET.
```
Coreeeeee

danroth27 commented:
I went through and cleaned out all of the codeplex links. 

I also removed the OData samples, since they are now maintained here: https://github.com/odata/odatasamples
